packages/advanced-logic/src/extensions/payment-network/address-based.ts

```diff
@@ -141,7 +142,36 @@ export default abstract class AddressBasedPaymentNetwork<
     };
   }
 
-  protected abstract isValidAddress(_address: string, _networkName?: string): boolean;
```

Comment:
I added a default implementation of the method to reduce code duplication in the extended classes.